cumulusci/robotframework/pageobject_mixin.py

```diff
@@ -0,0 +1,117 @@
+import re
+
+"""
+Page Object management functions and keywords
+
+The code in this mixin is for finding and loading the correct
+page object for the current page.
+
+"""
+
+
+class PageObjectMixin(object):
+    def current_page_should_be(self, name):
+        """Verifies that the current page matches the given page object
+
+        If the current page is what it's expected to be, the given page object
+        will become the new current page.
+        """
+        po = self._get_page_object(name)
+        if not po._is_current_page():
+            self.selenium.capture_page_screenshot()
+            raise Exception("The page '{}' is not the current page".format(name))
+        self.set_current_page_object(name)
+
+    def go_to_tab(self, name):
+        """Go to the tab associated with the given page object name"""
+        po = self._get_page_object(name)
+        po._go_to_tab()
+
+    def set_current_page_object(self, name):
+        """This moves the given page object library to the start of the search order
+
+        """
+        po = self._get_page_object(name)
+        self.builtin.log("setting page object to '{}'".format(str(po)), "DEBUG")
+
+        self.builtin.set_library_search_order(po.__class__.__name__)
```

Comment:
I've been going back and forth on that. I had code to do that and then I took it out right before pushing because it wasn't doing anything useful for the current demo. 

I'm glad you brought it up. I haven't made a final decision, but it's good to know that the idea occurred to you too.

Reply:
I don't have a strong feeling about it, but it we don't, we should probably at least document that.